classes/webservice/WebserviceRequest.php

```diff
@@ -1116,7 +1116,7 @@ protected function manageFilters()
                         if ($field != 'sort' && $field != 'limit') {
                             if (!in_array($field, $available_filters)) {
                                 // if there are linked tables
-                                if (isset($this->resourceConfiguration['linked_tables']) && isset($this->resourceConfiguration['linked_tables'][$field])) {
+                                if (isset($this->resourceConfiguration['linked_tables'], $this->resourceConfiguration['linked_tables'][$field])) {
```

Comment:
you can simplify with `isset($this->resourceConfiguration['linked_tables'][$field])`